cmd/minikube/cmd/start.go

```diff
@@ -87,8 +86,9 @@ var startCmd = &cobra.Command{
 
 // Set default value for host data and config dir
 var (
-	hostConfigDirectory = "/var/lib/minishift/openshift.local.config"
-	hostDataDirectory   = "/var/lib/minishift/hostdata"
+	hostConfigDirectory  = "/var/lib/minishift/openshift.local.config"
+	hostDataDirectory    = "/var/lib/minishift/hostdata"
+	hostVolumesDirectory = "/var/lib/minishift/openshift.local.volumes"
```

Comment:
+1. Good that we got the bottom of this. Let's get this implemented asap. We might take issue #409 into consideration, but we can defer this as well.